docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,167 @@
+| **Enterprise** | Custom pricing | For custom solutions, including deployment into virtual private clouds (VPCs) with dedicated technical support.  |
+
+Semgrep App integrates with a GitHub or GitLab account at either the organizational or individual level. There is no limit to the number of members in an organization for all tiers.
+
+Billing depends on the number of **seats**. Any member whose code has been scanned by Semgrep App takes up a seat.
+
+Members who do **not** run any scans on their code do not need a seat and are not billed. Semgrep Team tier features apply to the entire organization, even if not all members are billed for a seat.
+
+### Comparing free and paid tier features
+
+#### Organizational management
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- |
+| **Member limit** | ♾️ unlimited | ♾️  unlimited | ♾️  unlimited |
+| **Project limit** |♾️  unlimited | ♾️  unlimited | ♾️  unlimited |
+
+#### Findings, language support, and rules
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Findings retention** | 1 month | 1 year | 1 year+ (custom) |
+| **Community rule registry** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Named rules** | ✔️ yes | ✔️ yes | ✔️  yes |
+| **Private rules** | ❌ no | ✔️ yes | ✔️  yes |
+| **Custom rule messages** | ❌ no | ✔️  yes | ✔️  yes |
+| **Custom language support** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* **Named rules** are rules that have been saved through the [Playground](https://semgrep.dev/editor). The Playground is a live editor where you can write your rule patterns and test them on sample code. Named rules created by Community tier members are public and available in the Playground.
+* **Custom rule messages** can be applied to public and private rules.
+
+#### Deployment, integrations, and notifications
+
+| Feature  | Community | Team | Enterprise |
+| -------  | --------- | ------- | -------- | 
+| **Automatic CI/CD** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Slack notifications** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Email notifications** | ✔️ yes | ✔️  yes  | ✔️  yes | 
+| **Webhook integration** | ❌ no | ✔️  yes | ✔️  yes |
+| **Jira integration** | ❌ no | ✔️  yes | ✔️  yes |
+| **VPC deployment** | ❌ no | ❌ no | ✔️  yes |
+
+Notes:
+
+* There is no limit to the number of integrations or notification channels. You can make more than one integration of any type.
+* **Webhook integration** uses a generic webhook to send JSON messages. These messages are triggered when a policy is changed, a new scan runs, or a new finding has emerged.
+* **Jira integration** allows organizations to directly create Jira tickets from a finding.
+
+#### Authentication
+
+| Feature                                  | Community | Team | Enterprise |
+| ------                                    | --------- | ------- | -------- | 
+| **GitHub or GitLab single sign-on (SSO)** | ✔️ yes | ✔️  yes  | ✔️  yes | 
```

Comment:
Should we add a caveat that GitLab SSO is only available for users of GitLab.com, not for users of GitLab self-managed?

Reply:
Have added this caveat in. Thank you for the catch!